osu.Game/Rulesets/Scoring/HitWindows.cs

```diff
@@ -168,6 +182,29 @@ public double WindowFor(HitResult result)
         /// Defaults are provided but can be overridden to customise for a ruleset.
         /// </summary>
         protected virtual DifficultyRange[] GetRanges() => base_ranges;
+
+        public class EmptyHitWindows : HitWindows
+        {
+            private static readonly DifficultyRange[] ranges =
+            {
+                new DifficultyRange(HitResult.Perfect, 0, 0, 0),
```

Comment:
I think you meant `Great` here?
```suggestion
                new DifficultyRange(HitResult.Great, 0, 0, 0),
```

Reply:
I didn't, as I typed Perfect.

Can you explain why Great should be preferred?